tests/integration/routes/test_feedback.py

```diff
@@ -0,0 +1,134 @@
+from app import settings
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestFeedback(IntegrationTestCase):
+    def test_questionnaire_not_completed(self):
+        # Given I launch the test_feedback questionnaire
+        self.launchSurvey("test_feedback")
+
+        # When I try to view the feedback page without completing the questionnaire
+        self.get("/submitted/feedback/send")
```

Comment:
Given the `send` and `sent` URLs are used multiple times, I think they warrants a top level variable.

Reply:
Send makes send, sent only gets called once

Reply:
`sent` is used 4 times.